### PR TITLE
Duration prediction

### PR DESCRIPTION
**Pull request by emmanuelkyeremeh**
Completed the duration prediction project as part of the MLOps tutorial videos

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+env
+.ipynb_checkpoints
```
Comment (Silas-Asamoah) [inline .gitignore:2]: Kindly add binary files to your git ignore

File: Duration-prediction.ipynb
Comment (Silas-Asamoah) [inline Duration-prediction.ipynb:1]: This is a great start, but your workflow doesn't seem to incorporate data exploration/data science you went straight to developing the model. Although you had great results, I don't think you explored the data to understand what the different variables and components mean. Kindly incorporate this in your future assignments

**Review by Silas-Asamoah — APPROVED** [review]
Overall looks good! Nice stuff, left a few comments on this but good to merge @emmanuelkyeremeh

**Comment by Silas-Asamoah** [comment]
@emmanuelkyeremeh love the checks you've incorporated into the repository. Well done!

File: Duration-prediction.ipynb
Comment (emmanuelkyeremeh) [reply Duration-prediction.ipynb:1]: Duly noted

**Pull request merged**